plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/TestMySqlConfig.java

```diff
@@ -44,12 +45,14 @@ public void testExplicitPropertyMappings()
                 .put("mysql.max-reconnects", "4")
                 .put("mysql.connection-timeout", "4s")
                 .put("mysql.jdbc.use-information-schema", "false")
+                .put("mysql.enforce-millis-timestamp-precision", "true")
```

Comment:
`.put("mysql.enforce-timestamp-precision", "3")`?

Reply:
no, we don't want this to be more parameterized